### ISSUE TITLE
Support Escaped Composer Names

### ISSUE THREAD

**Issue by sprankhub** [yireo/Yireo_ExtensionChecker#35]
I just got an interesting output with the latest version 2.2.0:

    +-------------------------------------+-----------------------------+-------------------------------------+
    | Message                             | Group                       | Suggestion                          |
    +-------------------------------------+-----------------------------+-------------------------------------+
    | No composer dependency found for "" | Missing composer dependency | Current version is . Perhaps use ~? |
    +-------------------------------------+-----------------------------+-------------------------------------+

Debugging showed that the module, which was (correctly) detected as a dependency, was disabled in `app/etc/config.php`. It is correct that the extension checker reports this as an issue, but it should name the module. Should we maybe fall back to the `$moduleName` here?

https://github.com/yireo/Yireo_ExtensionChecker/blob/9c3d201fee9039b1ab9cae8823d4b4ab442b5759/Component/ComponentFactory.php#L49

**Comment by jissereitsma** [comment]
No, I'd rather keep this here as it is, but the message that is outputted is annoying, so I'll just change it there: https://github.com/yireo/Yireo_ExtensionChecker/commit/76402528711648dc00020c352b2af879d420d767 

However, the other intruiging part is why modules that are disabled are not detected. The reason is because in many places the core `ModuleInfo` class only picks up upon enabled modules. But on the other hand, I started to create a class `Yireo\ExtensionChecker\Util\OtherModuleInfo` (great name) already to overcome this issue. This will require quite some refactoring I guess - too much to quickly do right now. Let's keep this issue open for reminders.
